curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/accessing-object-properties-with-variables.english.md

```diff
@@ -40,7 +40,7 @@ Note that we do <em>not</em> use quotes around the variable name when using it t
 
 ## Instructions
 <section id='instructions'>
-Use the <code>playerNumber</code> variable to look up player <code>16</code> in <code>testObj</code> using bracket notation. Then assign that name to the <code>player</code> variable.
+Set the <code>playerNumber</code> variable to number <code>16</code>. You should then assign player 16's name to the <code>player</code> variable by looking up player 16's name in <code>testObj</code> with bracket notation and the <code>playerNumber</code> variable.
```

Comment:
```suggestion
Set the <code>playerNumber</code> variable to <code>16</code>. Then assign player 16's name to the <code>player</code> variable by looking it up in <code>testObj</code> using bracket notation and the <code>playerNumber</code> variable.
```

I think this looks pretty good @bhagerty. I agree that this one is confusing without looking at the test text. I made a suggestion to try and make the instructions a little more concise. Perhaps I'm being too picky.

Reply:
@bhagerty If you are fine with @moT01's, you can simply click the `Commit suggestion` button above.  I like Tom's suggestion and I believe it accomplishes what you were trying to do in the first place.